pkg/controller/scyllacluster/resource_test.go

```diff
@@ -359,11 +369,46 @@ func TestMemberService(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:          "last cleaned up annotation is added when cleanup job is complete",
```

Comment:
could also use a test to make sure it stays there if it was on the old service

Reply:
done